eplaunch/workflows/builtin/workflows/app_g_postprocess.py

```diff
@@ -0,0 +1,16 @@
+from eplaunch.workflows.base import BaseWorkflow
+
+
+class AppGPostProcessWorkflow(BaseWorkflow):
+
+    def name(self):
+        return "AppGPostProcess"
+
+    def description(self):
+        return "Run Appendix G Post-processor"
+
+    def get_file_types(self):
+        return ["*.html"]
+
+    def get_extra_data(self):
+        return {"Hey, it's extra": "data"}
```

Comment:
So I created all these "builtin" workflow files.  Ultimately, EP-Launch-3 shouldn't come with any, or well, it shouldn't come with these specific E+ ones.  These should come with E+, and be tailored to each specific version of E+.  But for now, they are "built-in".  We can decide if we do want to include some generic ones (File Operations anyone?).

Reply:
Agreed